quiz.js

```diff
@@ -7,26 +7,22 @@ let quizTime = 0;
 let numWrong = 0;
 let numCorrect = 0;
 
-for (i = 0; i < 3; i++) {
+for (let i = 0; i < questions.length; i++) {
     let difficulty = 1;
-    tstart = new Date().getTime();
     //console.log(tstart);
-    let userAnswer = prompt(questions[i].question);
-    if (typeof(userAnswer) === "string") {
-        userAnswer = userAnswer.toLowerCase();
-    }
+    let userAnswer = prompt(questions[i].question).toLowerCase();
     if (userAnswer === questions[i].answer) {
         console.log('Correct! The answer is',questions[i].answer);
         numCorrect++;
         difficulty++;
-        } else {
+    } else {
         console.log('Sorry',userAnswer,'is incorrect. The correct answer is',questions[i].answer,'.')
         numWrong++;
     }
     tend = new Date().getTime();
     //console.log(tend);
-    quizTime = quizTime + (tend-tstart);
 }
+quizTime = tend-tstart;
```

Comment:
Moving the quiz time down here will make the code a bit more efficient. Right now you are doing a time calculation for each question. With this change your time calculation will only be done once at the end so it's just fewer operations for your code to have to run.